rel-eng/tito.props

```
@@ -499,6 +499,7 @@ whitelist = nodejs-babel-plugin-module-resolver
 disttag = .el7
 whitelist = ansiblerole-foreman_scap_client
   ansiblerole-insights-client
+  ansiblerole-satellite-receptor-plugin
```

```suggestion
  ansiblerole-satellite-receptor-installer
```

The error you pasted is not the error you're looking for.

The real problem is that you did whitelist the wrong package name (why do we have so many names anyways?).